src/Display/Display.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+// This is a div element that is in font-size bigger than h1 and is supposed to
+// used as 'slogan' around our apps. We dont want this to be crawled from search engines
+// since its only for design reasons but instead we want the h1 to be so as it is meant by default.
+const DisplayStyles = styled.div`
+  font-weight: 500;
+  line-height: 1.13;
```

Comment:
Where did you get this lineheight value from? seems should be either 1.16 or 56px?

Reply:
ehm when you click on the element from Zeplin then you can see the styling of the component. This is where I take the styles from always.

Reply:
Ahhhhhh, I see. Weird its different to what the actual text says right next to it. Can you check with piotr which one is correct?

Reply:
rem sounds the most correct.

Reply:
Can you explain why? Also does that mean we should be using rem for font size?

Reply:
i dont have a strong reason I think out of the top of my mind that rem is more responsive. 
I will convert to pixel you are right.

Reply:
Take some time and look into it maybe we should be using that here instead, not against it just need some reasoning

Reply:
uhuh will do then.